Author: Christian Inci <chris.gh@broke-the-inter.net>
Date:   Mon May 15 21:26:05 2023 +0200
Upstream: https://github.com/gnuradio/gnuradio/pull/6667

    Include cstdint in gr-fec's alist.h
    
    GCC 13 will error out otherwise.
    
    Signed-off-by: Christian Inci <chris.gh@broke-the-inter.net>

--- a/gr-fec/include/gnuradio/fec/alist.h
+++ b/gr-fec/include/gnuradio/fec/alist.h
@@ -23,6 +23,7 @@
 #define ALIST_H
 
 #include <gnuradio/fec/api.h>
+#include <cstdint>
 #include <cstdlib>
 #include <fstream>
 #include <sstream>
